### ISSUE TITLE
collapsing to unique 'x' values

### ISSUE THREAD

**Issue by MLopez-Ibanez** [auto-optimization/iraceplot#13]
Warning generated by `boxplot_performance()`
```
#> Warning in regularize.values(x, y, ties,
#> missing(ties), na.rm = na.rm): collapsing to
#> unique 'x' values
```

**Comment by MLopez-Ibanez** [comment]
OK, this will be fixed in ggplot2 3.3.6 https://github.com/tidyverse/ggplot2/pull/4654

**Comment by MLopez-Ibanez** [comment]
We now require ggplot 3.3.6.
